mono/metadata/file-mmap-posix.c

```diff
@@ -480,7 +490,9 @@ mono_mmap_configure_inheritability (void *mmap_handle, gint32 inheritability, Mo
 	int fd, flags;
 
 	fd = h->fd;
+	MONO_ENTER_GC_SAFE;
 	flags = fcntl (fd, F_GETFD, 0);
```

Comment:
Does this do I/o or just set flags?
Granted there could be a lock around those flags, but it should be brief, unles its holder is otherwise busy.

Reply:
I'm pretty sure this is one of the ones where I checked to make sure it used a lock internally. Not sure how brief that would be and whether it's worth the transition, so I included it for now.